src/privateai_client/components/pai_responses.py

```diff
@@ -116,6 +116,14 @@ def entities(self):
     def entities_present(self):
         return self.get_attribute_entries("entities_present")
 
+    @property
+    def best_labels(self):
+        if type(self.body) == dict:
+            best_labels = [entity["best_label"] for entity in self.entities]
+        else:
+            best_labels = [attr["best_label"] for entity in self.entities for attr in entity]
+        return best_labels
+
```

Comment:
[Q] Are there tests required for this?

Reply:
Would be nice but we don't have any response tests right now. Wiring up some mocks or better yet, full integration tests would be sweet though